lib/plugins/slack/attachements.js

```diff
@@ -11,7 +11,7 @@ function getMetric(metric) {
   }
 }
 
-module.exports = function(dataCollection, slackOptions) {
+module.exports = function(dataCollection, slackOptions, resultBaseURL, storageManager) {
```

Comment:
I think it'd be good to place the slackOptions map last (that's the pattern we use for other functions that take options).